### ISSUE TITLE
Birthdate error

### ISSUE THREAD

**Issue by qwerm14** [NightfallGT/Discord-Account-Generator#16]
it doesnt put the birthdate in for me i have to do it myself??

**Comment by epicmanmoo** [comment]
Yes, if the program can't find the appropriate XPATH for the date selectors, you have to enter it manually.

**Comment by 1337-69420** [comment]
> @qwerm14 Yes, if the program can't find the appropriate XPATH for the date selectors, you have to enter it manually.

the program is not programmed right, it never finds the xpath lmao L

**Comment by 1337-69420** [comment]
I fixed it btw but it is not worth to send a pull request because all this does is produce verification locked accounts

**Comment by freedomshare** [comment]
@1337-69420  can you please tell me how to fix birth date also your locked account might becuase of the ip you are using

**Comment by epicmanmoo** [comment]
> > @qwerm14 Yes, if the program can't find the appropriate XPATH for the date selectors, you have to enter it manually.
> 
> the program is not programmed right, it never finds the xpath lmao L

Discord changes its UI periodically, that is why the code does not always work. Some UIs are also different depending on the proxies you are using (the US vs international proxies).
